Dockerfile

```diff
@@ -1,17 +1,27 @@
-FROM golang:1.16 AS build
```

Comment:
we still want this dockerfile since this is for removing images. You can name it `Dockerfile.eraser` or something to differentiate

Reply:
and obviously change `example`

Reply:
@ashnamehrotra can you make sure to add the Dockerfile for eraser

Reply:
should be under pkg/eraser as Dockerfile.eraser